### PR TITLE
Milestone 5 - dynamic page

### PR DESCRIPTION
**Pull request by Khanetmoi**
Capstone : Desktop
   - Implement the section "Featured speakers" with dynamic HTML.
   - Deploy the project and check to make sure everything works as planned.

video link  [my video](https://www.loom.com/share/cdcecb86d0ca4cc187f6efd74a2da17b)

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Khanetmoi   😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Video presentation is missing. Kindly record a loom video that contains the following and add the link in the README file.
 - Record a video presenting the features of the project.
 - The maximum length of the video is 5 minutes.
 - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
 - The video contains a demo of the project, showing the pages in different screen sizes.
 - The video contains a highlight about a piece of the code that you find interesting or are proud of making.

- [ ] Please use the requested Font: Fonts should be used: Cocogoose, Lato
 
- [ ] The pages should look almost identical to the original design, outside the content differences.
#### [Required Design]
Screen Shot 2022-05-19 at 5 11 07 AM

#### [Your Design]

Screen Shot 2022-05-19 at 5 12 47 AM

- [x] Kindly left-align the paragraph to make it look similar to the required design 👍🏻 
Screen Shot 2022-05-19 at 5 13 56 AM

- [x] Kindly work on the design.
#### [Required Design]
Screen Shot 2022-05-19 at 5 15 29 AM

#### [Your Design]
- Some icons are missing and others are not using the required color. Kindly fix it.
Screen Shot 2022-05-19 at 5 16 57 AM

#### [Rquired Design]
Screen Shot 2022-05-19 at 5 17 17 AM

#### [Your Design]
- There should be only two columns in the desktop version and there should be a checkbox image pattern behind each speaker's image. kindly fix this 👍🏻 
Screen Shot 2022-05-19 at 5 17 56 AM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — CHANGES_REQUESTED** [review]
# Required Changes
Hi @Khanetmoi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark:  Passing Linters
:heavy_check_mark:  Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] Please add video link to your PR description

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
-   - set up a new repository and prepare it for development using best practices .
-   - build the first 2 sections of the mobile website using the template 4 of the Microverse portfolios.
+    
+    - Implement the section "Featured speakers" with dynamic HTML.
+    - Deploy the project and check to make sure everything works as planned.
+
+# Acknowledgements
+   - Thanks to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+# Video Presentation
```
Comment (danmainah) [inline README.md:15]: - [ ] [OPTIONAL] In the future I would recommend installing Krisp it gives your every week to record clear sound and removes the background noise, it works very well with zoom and loom.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            <a href="#">
+                <div class="shamburger">
+                    <span class="bars"></span>
+                    <span class="bars"></span>
+                    <span class="bars"></span>
+                </div>
+            </a>
+            <ul class="disappears" id="navList2">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a> </li>
```
Comment (danmainah) [inline about.html:23]: - [ ] [OPTIONAL] It would be nice to remove the about page link when you are already in the about page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            <a href="#">
+            <div class="hamburger">
+                <span class="bars"></span>
+                <span class="bars"></span>
+                <span class="bars"></span>
+            </div>
+            </a>
+            <ul class="disappear" id="navList">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a> </li>
```
Comment (danmainah) [inline index.html:23]: - [ ] [OPTIONAL] It would be nice to remove the home page link when you are already on the home page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        </nav>
+    </header>
+    <section id="About">
+        <h1>Global Students Network</h1>
+        <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with students from all
+        over the world</p>
+        <h2 id="hot-date">2022.12.20(tue) - (fri)</h2>
+        <h3 id="event">@student exchange</h3>
+    </section>
+    <section id="programs">
```
Comment (danmainah) [inline index.html:58]: - [ ] As previously requested kindly ensure this section has only five programs as shown In the design given, kindly remove the extra projects and also make sure the design matches the given design

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Khanetmoi , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Khanetmoi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**